src/main/java/gregtech/integration/jei/recipe/RecipeMapCategory.java

```diff
@@ -106,15 +100,15 @@ public void setRecipe(IRecipeLayout recipeLayout, @Nonnull GTRecipeWrapper recip
                 if (handle.getItemHandler() == importItems) {
                     //this is input item stack slot widget, so add it to item group
                     itemStackGroup.init(handle.getSlotIndex(), true,
-                            new ItemStackTextRenderer(notConsumedItemInputs.get(handle.getSlotIndex())),
+                            new ItemStackTextRenderer(recipeWrapper.isNotConsumedInput(handle.getStack())),
                             slotWidget.getPosition().x + 1,
                             slotWidget.getPosition().y + 1,
                             slotWidget.getSize().width - 2,
                             slotWidget.getSize().height - 2, 0, 0);
                 } else if (handle.getItemHandler() == exportItems) {
                     //this is output item stack slot widget, so add it to item group
                     itemStackGroup.init(importItems.getSlots() + handle.getSlotIndex(), false,
-                            new ItemStackTextRenderer(chanceOutputMap.get(importItems.getSlots() + handle.getSlotIndex())),
+                            new ItemStackTextRenderer(recipeWrapper.hasChancedOutput(handle.getStack())),
```

Comment:
This is not working

Reply:
As above - probably not the correct ItemStack
Also, this needs to be passed a ChanceEntry, since the renderer needs to know what the chance is to be able to render it.  If you pass it a boolean, it will assume you wanted to specify whether the item is not consumed